cpp/ql/src/semmlecode.cpp.dbscheme

```diff
@@ -1190,6 +1190,25 @@ funbind(
                     | @delete_expr
                     | @delete_array_expr;
 
+
+@prefix_crement_oper_expr = @preincrexpr | @predecrexpr
+
+@postfix_crement_oper_expr = @postincrexpr | @postdecrexpr
+
+@incr_oper_expr = @preincrexpr | @postincrexpr
+
+@decr_oper_expr = @predecrexpr | @postdecrexpr
+
+@crement_op_expr = @incr_oper_expr | @decr_oper_expr | @prefix_crement_oper_expr | @postfix_crement_oper_expr
```

Comment:
I find the naming of the five crement unions to be inconsistent wrt. what's spelled out and what's abbreviated. Should it be `op` or `oper`? I think I prefer neither for the crement expressions. How about `@{prefix,postfix}_crement_expr`, `@{increment,decrement}_expr`, and `@crement_expr`?

Reply:
I like your renamings... and I agree that it was not very consistent! I've pushed a new .dbscheme file with the required changes